style.css

```diff
@@ -546,20 +569,44 @@ hr#last {
 
 /* Media Query */
 
-/* For Tablets And Ipads */
-@media screen and (max-width: 992px) {
-  html {
-    font-size: 55%;
-  }
-}
-
 /* For Smarth phones */
-@media screen and (min-width: 280px) and (max-width: 768px) {
+@media screen and (min-width: 375px) and (max-width: 768px) {
```

Comment:
- [ ] As suggested by the previous reviewer and based on the project requirement, please make sure the desktop version usually starts from `768px`, You already have done the mobile version of your portfolio so you don't need media query for the mobile version.  Currently it is `<= 768`. Kindly, change it.
![image](https://github.com/ZabihullahNooriWardak/Zabih-portfolio/assets/101523582/c266c455-9a4c-44da-8c52-a3b33aa8c1ea)

Therefore, you need to change this line to the following code and use the necessary property of `CSS` like `Grid` and `Flexbox` to make it responsive
`@media screen and (min-width: 768px) {
  /* CSS styles for screens with a minimum width of 768px (desktop and tablets) */`

Your design
![image](https://github.com/ZabihullahNooriWardak/Zabih-portfolio/assets/101523582/b7fdfba0-4089-45af-ae67-891492bd241e)

Figma design
![image](https://github.com/ZabihullahNooriWardak/Zabih-portfolio/assets/101523582/934f7227-df58-4ae6-9432-7499791e740f)

- [ ] As suggested by the previous reviewer, please make sure your design is the same as the given Figma design.